scripts/template_upload_scvmm.py

```diff
@@ -35,65 +36,66 @@ def parse_cmd_line():
     return args
 
 
-def check_template_exists(client, name):
-    if name in client.list_template():
-        return True
-    else:
-        return False
-
-
 def upload_vhd(client, url, library, vhd):
 
-    print("SCVMM: Downloading VHD file, then updating Library")
+    logger.info("SCVMM: Downloading VHD file, then updating Library")
```

Comment:
I understand.  Maybe we should implement logger.print() which would do both.  I'll put those back.